README.md

```diff
@@ -65,7 +64,7 @@ metadata:
   name: kubesec-demo
 spec:
   containers:
-  - name: kubesec-demo
+  * name: kubesec-demo
```

Comment:
This is YAML and should remain a `-` please :pray:

Reply:
oops.

In response to your preference for dashes, I can swap it all back if you prefer but then w have to  either change the CoC to use Dashes or ignore it from linting